### PR TITLE
Remove zfs usage in Xenial

### PR DESCRIPTION
**Pull request by natefinch**
Fixes https://bugs.launchpad.net/juju/+bug/1648513

### CODE REVIEW

File: container/lxd/initialisation_linux.go [hunk 1]
```diff
@@ -31,9 +31,7 @@ var requiredPackages = []string{
 	"lxd",
 }
 
-var xenialPackages = []string{
-	"zfsutils-linux",
-}
+var xenialPackages = []string{}
```
Comment (bz2) [inline container/lxd/initialisation_linux.go]: Remove this var and the loop over it in `ensureDependencies()`?

File: container/lxd/initialisation_linux.go [hunk 2]
```diff
@@ -68,7 +64,14 @@ func (ci *containerInitialiser) Initialise() error {
 	// Well... this will need to change soon once we are passed 17.04 as who
 	// knows what the series name will be.
 	if ci.series >= "xenial" {
```
Comment (bz2) [inline container/lxd/initialisation_linux.go]: One query is whether skipping lxd init on older series is still correct.

Comment (natefinch) [inline container/lxd/initialisation_linux.go]: ¯\_(ツ)_/¯

**Comment by natefinch** [comment]
$$merge$$

**Pull request merged**